dask/array/tests/test_slicing.py

```diff
@@ -473,19 +473,19 @@ def test_slicing_exhaustively():
     # independent indexing along different axes
     indexers = [0, -2, I[:], I[:5], [0, 1], [0, 1, 2], [4, 2], I[::-1], None, I[:0], []]
     for i in indexers:
-        assert_eq(x[i], a[i]), i
+        assert_eq(x[i], a[i])
```

Comment:
From what I understand, this tuple expression was indeed useless, and the second element was ignored.